charts/data-service/values.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Crown Copyright
```

Comment:
Have not figures out to select a file that has not been edited, but I did find a file in this branch that has been updated with the new copyright header.  It is the README.md file in the resource-service